app/views/refinery/pages/home.html.erb

```diff
@@ -39,7 +39,6 @@
           <% end %>
         </div>
         <div class="content-column-two">
-
```

Comment:
While I agree with this minor update I generally prefer to keep things where we prettify to a separate PR to avoid misdirection when perusing git history.